cas/service.c

```diff
@@ -1052,29 +1053,28 @@ static int cas_dtm0_logrec_credit_add(struct m0_fom *fom0)
 	return M0_RC(rc);
 }
 
-static int cas_dtm0_logrec_add(struct m0_fom *fom0, struct m0_dtm0_tx_desc *txd,
+static int cas_dtm0_logrec_add(struct m0_fom *fom0,
 			       enum m0_dtm0_tx_pa_state state)
 {
 	/* log the dtm0 logrec before completing the cas op */
 	struct m0_dtm0_service *dtms =
 		m0_dtm0_service_find(fom0->fo_service->rs_reqh);
 	struct m0_dtm0_tx_desc *msg = &cas_op(fom0)->cg_txd;
-	struct m0_buf	        buf = {};
+	struct m0_buf           buf = {};
 	int                     i;
-	int			rc;
+	int                     rc;
 
 	for (i = 0; i < msg->dtd_ps.dtp_nr; ++i) {
 		if (m0_fid_eq(&msg->dtd_ps.dtp_pa[i].p_fid,
-					&dtms->dos_generic.rs_service_fid)) {
-			msg->dtd_ps.dtp_pa[i].p_state =
-				(uint32_t)M0_DTPS_PERSISTENT;
+			      &dtms->dos_generic.rs_service_fid)) {
+			msg->dtd_ps.dtp_pa[i].p_state = state;
 			break;
 		}
 	}
 	rc = m0_xcode_obj_enc_to_buf(&M0_XCODE_OBJ(m0_cas_op_xc, cas_op(fom0)),
 				     &buf.b_addr, &buf.b_nob) ?:
-		m0_dtm0_logrec_update(dtms->dos_log, &fom0->fo_tx.tx_betx, msg,
-				      &buf);
+	     m0_dtm0_logrec_update(dtms->dos_log, &fom0->fo_tx.tx_betx, msg,
```

Comment:
Please check indent here (possibly a github's issue).

Reply:
Not sure what should be the indentation here. Will leave it at the same level as before.